services/object/src/api.js

```diff
@@ -10,4 +11,25 @@ let builder = new APIBuilder({
   context: ['cfg', 'db', 'backends'],
 });
 
+builder.declare({
+  method: 'post',
+  route: '/upload/:name/:projectId',
+  name: 'uploadObject',
+  input: undefined,
+  output: undefined,
+  stability: 'experimental',
+  category: 'Backend',
```

Comment:
```suggestion
  category: 'Upload',
```
rationale: we'll probably have 2-3 methods for uploading objects, and 2-3 methods for downloading.  A user of the API will probably only be concerned with one or the other of those.  So those will make nice categories.